vpdq/cpp/regtest.py

```diff
@@ -8,7 +8,13 @@ def get_argparse() -> argparse.ArgumentParser:
     ap = argparse.ArgumentParser(
         description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
```

Comment:
Any reason to use RawDescriptionHelpFormatter? This is usually if you are doing custom things with indenting

Reply:
Research the formatter_class a little bit, decide to remove it.